README.md

```diff
@@ -28,24 +28,24 @@ by the user.
 Working with parameters is done through the `CLAide::ARGV` class. It takes an
 array of parameters and parses them as either flags, options, or arguments.
 
-| Parameter             | Description                                       |
-| :---:                 | :---:                                             |
-| `--milk`, `--no-milk` | A boolean ‘flag’, which may be negated.           |
-| `--sweetner=honey`    | A ‘option’ consists of a key, a ‘=’, and a value. |
-| `tea`                 | A ‘argument’ is just a value.                     |
+| Parameter              | Description                                        |
+| :---:                  | :---:                                              |
+| `--milk`, `--no-milk`  | A boolean ‘flag’, which may be negated.            |
+| `--sweetener=honey`    | An ‘option’ consists of a key, a ‘=’, and a value. |
+| `tea`                  | An ‘argument’ is just a value.                     |
 
 
 Accessing flags, options, and arguments, with the following methods, will also
 remove the parameter from the remaining unprocessed parameters.
 
 ```ruby
-argv = CLAide::ARGV.new(['tea', '--no-milk', '--sweetner=honey'])
+argv = CLAide::ARGV.new(['tea', '--no-milk', '--sweetener=honey'])
 argv.shift_argument     # => 'tea'
```

Comment:
the indentation of these comments needs to be adjusted (just 1 space to the right, I think)